devices/esp32-c3/main/build/log/idf_py_stderr_output_4114

```diff
@@ -0,0 +1,19 @@
+CMake Warning (dev) in CMakeLists.txt:
```

Comment:
lets add /log to gitignore?

Reply:
I deleted the whole build directory and added that to .gitignore